modules/platforms/dotnet/build.ps1

```diff
@@ -195,6 +195,18 @@ if (!$skipDotNet) {
 	echo "Starting MsBuild: '$msBuildCommand'"
 	cmd /c $msBuildCommand
 
+    # Build core
```

Comment:
Here and below we override the exit code from Classic .NET. Let's move that check higher, so that MSBuild failure exits the script as before.